app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -312,7 +311,8 @@ open class AccountStore(
 
         this.generateNewFirefoxAccount()
 
-        webView.clearCache(true)
+        WebView(context).clearCache(true)
```

Comment:
Maybe add a comment here to explain why we're doing this? I think that might help clear up the ambiguity